wradlib/georef/raster.py

```diff
@@ -300,13 +302,13 @@ def set_raster_origin(data, coords, direction):
         coords = np.flip(coords, axis=-3)
         # we need to shift y-coordinate if data and coordinates have the same
         # number of rows and cols (only the ll or ul raster coords are given)
-        if data.shape[-2:] == coords.shape[:2]:
-            coords += [0, y_sp]
+#        if data.shape[-2:] == coords.shape[:2]:
```

Comment:
Here too, pixel corner coordinates are assumed. If we only have the upper left coordinates of the pixels we need to shift the coordinates after flipping because upper left will then be lower left.

Reply:
In this case center coordinates are easier ;-)

Reply:
Shifting is not needed any more, since we have now proper alignment . Just delete the remnants.